python/CMakeLists.txt

```diff
@@ -1,9 +1,29 @@
+cmake_minimum_required(VERSION 3.12)
+
+project(pyf3d)
+
+find_package(Python 3.6 COMPONENTS Interpreter Development)
+message(STATUS "Bindings: Python: Python ${Python_VERSION} found")
+
 find_package(pybind11 2.2 REQUIRED)
 message(STATUS "Bindings: Python: pybind11 ${pybind11_VERSION} found")
 
+# In case pyf3d is built separately, we need to retrieve the existing libf3d
+if(NOT TARGET libf3d)
+  find_package(f3d REQUIRED)
+  message(STATUS "Bindings: Python: libf3d ${f3d_VERSION} found")
+
+  set(_f3d_include_path "${f3d_INCLUDE_DIR}/f3d")
+else()
+  add_library(f3d::libf3d ALIAS libf3d)
+  set(_f3d_include_path "${CMAKE_SOURCE_DIR}/library/public")
+endif()
+
 pybind11_add_module(pyf3d F3DPythonBindings.cxx)
 
-target_link_libraries(pyf3d PRIVATE libf3d)
+target_include_directories(pyf3d PRIVATE "${_f3d_include_path}" )
```

Comment:
this was not needed before strangely

Reply:
It's not strictly needed for local build, because libf3d target has public includes to `library/public`, but for external build the target has public includes to `/usr/include` and the headers are in `/usr/include/f3d`.  
Alternatively, we could move all headers to `library/public/f3d` and use `#include <f3d/xxx.h>` everywhere.

Reply:
as you see fit. ok as is.